Documentation/operations/upgrade.rst

```diff
@@ -1382,10 +1402,10 @@ The cilium preflight manifest requires etcd support and can be built with:
       --namespace=kube-system \
       --set preflight.enabled=true \
       --set agent.enabled=false \
-      --set config.enabled=false \
+      --set enabled=false \
       --set operator.enabled=false \
-      --set global.etcd.enabled=true \
-      --set global.etcd.ssl=true \
+      --set etcd.enabled=true \
+      --set etcd.ssl=true \
```

Comment:
this should not be modified since this section referes to upgrade from 1.7 to 1.8, we aren't changing the helm chart of 1.8 branch.

Reply:
fixed